src/M6Web/Bundle/DaemonBundle/Tests/Units/Command/DaemonCommandConcreteIterationCallback.php

```diff
@@ -29,6 +29,6 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
     protected function myCallback(InputInterface $input, OutputInterface $output)
     {
-        $this->countCall++;
+        ++$this->countCall;
```

Comment:
what's the point of changing this?
it doesn't change anything IMHO, since we don't use any `return` statement here